src/Toggle/ToggleChecker.php

```diff
@@ -51,9 +46,28 @@ public function isToggleActive( $toggleName )
         if ( !$this->toggleGateway->isToggleVisibleForUsers($toggleName) ) {
             return false;
         }
-        $groupPolicyResponse = $this->groupPolicy->getTogglePolicy($toggleName, $this->group);
+        $groupPolicyResponse = $this->transformByType(
+            $this->groupPolicy->getTogglePolicy($toggleName, $this->group),
+            $this->toggleGateway->getType($toggleName));
         return $groupPolicyResponse->isEnabled() ||
```

Comment:
In my understanding, when the toggle type is 'simple', then users should be able to override group settings if they want to.